dali/base/dafdesc.cpp

```diff
@@ -3210,17 +3210,20 @@ void GroupInformation::createStoragePlane(IPropertyTree * storage, unsigned copy
     StringBuffer mirrorname;
     const char * planeName = name;
     if (copy != 0)
-        planeName = mirrorname.append(name).append("_mirror").append(copy);
+        planeName = mirrorname.append(name).append("_mirror");
```

Comment:
should this perhaps append copy if >1 ? in case createStoragePlane ever called like that?

Reply:
I think revisit if that ever happens.  These "mirror" storage planes are currently unused.